### ISSUE TITLE
Node.js support range declared is incorrect

### ISSUE THREAD

**Issue by xfq** [validator/validator#2113]
### Summary

Node.js support range declared is incorrect.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Steps to reproduce

1. Go to https://github.com/validator/validator/blob/main/package.json

### Expected behavior

The `engines` field needs to reflect the actual minimum requirement, maybe at least Node.js 18.

https://github.com/validator/validator/blob/main/vnu-java-downloader.js uses stream/promises, async/await, etc.

### Actual behavior

https://github.com/validator/validator/blob/ff14d46adccb6a984fe4cf5835c169b89f021db0/package.json#L7-L9
